hls4ml/converters/PyTorch/PyTorchPooling.py

```diff
@@ -0,0 +1,90 @@
+import math
+from hls4ml.converters.pytorch_to_hls import pytorch_handler
+from hls4ml.converters.utils import * #Get parse data format and the two padding functions 
+
+#Assuming version 0.5.0 not in master branch yet
+#In Progress, do NOT commit to master yet
+pooling_layers = ['MaxPool1D', 'MaxPool2D', 'AvgPool1D', 'AvgPool2D']
+@pytorch_handler(*pooling_layers) #Iterate through the 4 types of pooling layers
+
+def parse_pooling_layer(pytorch_layer, input_names, input_shapes, data_reader, config):
+    
+    """"
+    Suppose model has attribute pool1, model.pool1 = maxpool2D(a,b,c)
+    pytorch_layer = model_name.pool1, model_name.pool2, etc
+    """
+    layer = {}
+    
+    layer['name'] = layer_name # i.e. model.pool would give pool
+    layer['class_name'] = pytorch_layer['class_name'] # i.e. maxpool2D, maxpool1d, etc
+    layer['data_format'] = 'channels_first' #By Pytorch default, cannot change
+    
+    ### Unsure about what this does ###
+    layer['inputs'] = input_names 
+    
+    #Check if 1D or 2D. 1, 2 is the second to last character
+    if int(layer['class_name'][-2]) == 1:
+        
+        '''Compute number of channels'''
```

Comment:
I don't think you need this. Can you replace this with a comment? (like using #)

Reply:
Doc string is supposed to be used at the beginning of the function.